src/collector/sort_key/order.rs

```diff
@@ -346,3 +424,32 @@ where
             .convert_segment_sort_key(sort_key)
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::schema::OwnedValue;
+
+    #[test]
+    fn test_mixed_ownedvalue_compare() {
+        let u = OwnedValue::U64(10);
+        let i = OwnedValue::I64(10);
+        let f = OwnedValue::F64(10.0);
+
+        let nc = NaturalComparator::default();
```

Comment:
somehow the suggestion UI has been removed from github?

anyway can we change this to 

`let nc = NaturalComparator;`

(just to please clippy. I personally don't think it matters)